src/views/pages/NewMessage/Index.js

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PageTitle from '../../components/PageTitle';
 import Text from '../../components/Text';
 import InboxSubNav from '../../components/InboxSubNav';
 
 function NewMessage() {
+
+  const [message, setMessage] = useState({
+    message: '',
+    public_key: '',
+    signature: '',
```

Comment:
@Romulus10 We can probably just pull values from data store and insert here.

Reply:
We probably could. Signatures are on each message so we'd just need to rig it up to drop the signature into memory when you go to send it.